main/main.js

```diff
@@ -277,7 +303,7 @@ function createWindowWithBounds (bounds) {
   so registering a handler causes events to happen twice.
   See: https://github.com/electron/electron/issues/18322
   */
-  if (process.platform === 'win32') {
+  if (isPlatform('win32')) {
```

Comment:
Why here are you using `isPlatform('win32')` but in other changes `process.platform === PLATFORM_WIN`?

Reply:
I just commuted options 😉

I just say, we can use this very clear....

Use function or constant or enum 😁